### DISCUSSION TITLE
linkActiveClass in Nuxt3

### DISCUSSION THREAD

**Discussion by aslero** [nuxt/nuxt#16002] [Questions]
How to replace the active class on nuxt-link?

```
router: {
    linkActiveClass: 'active-link'
  },
```

**Comment by lihbr** [comment]
Hey there, really sorry for the late reply, but posting it at least for other people.

You can overwrite `<NuxtLink>` active class in two ways:
- Locally, using the `active-class` prop (similarly with `exact-active-class`, like with a `<RouterLink />`
- Globally, by defining your own `<NuxtLink>` component, see this section in the doc: https://nuxt.com/docs/api/components/nuxt-link#overwriting-defaults

This has been done so that libraries relying on the default active class wouldn't break while still allowing you to have a `<NuxtLink />` with your preferred defaults :)
